modules/file-system/pre-existing-network-storage/README.md

```diff
@@ -45,6 +45,21 @@ were directories. This is important when working with files that were created by
 another source, but there may have performance impacts. The `_netdev` mount option
 denotes that the storage device requires network access.
 
+The following is an example of using `pre-existing-network-storage` with the `lustre`
+filesystem:
+
+```yaml
+- id: lustrefs
+  source: modules/file-system/pre-existing-network-storage
+  settings:
+    fs_type: lustre
+    server_ip: 192.168.227.11@tcp
+    local_mount: /data
```

Comment:
nit: Generally we tend to use lustre as a scratch filesystem. It might be a better example to use `/scratch` as a local mount.

Reply:
Thanks for the tip, I've updated it to use `/scratch` for the `local_mount` variable.